website/app/styles/pages/foundations/typography.scss

```diff
@@ -4,7 +4,11 @@
  */
 #show-content-foundations-typography{
     .doc-typography-demo-display-list{
-        list-style-type: none;
         padding:0;
+        list-style-type: none;
+    }
+
+    .list-item-gap{
```

Comment:
```suggestion
    .list-item-gap {
```

Reply:
To this point, I should have the class name reflect what this is doing more specifically to this: `typography-list-item-gap`. Making updates locally then passing it through.